### PR TITLE
chore: automatically release nightly version from master

### PR DESCRIPTION
**Pull request by ZauberNerd**
<details>
<summary>Bors merge bot cheat sheet</summary>

We are using [bors-ng](https://github.com/bors-ng/bors-ng) to automate merging of our pull requests. The following table provides a summary of commands that are available to reviewers (members of this repository with push access) and delegates (in case of `bors delegate+` or `bors delegate=[list]`).

| Syntax | Description |
| --- | --- |
| bors merge | Run the test suite and push to master if it passes. Short for "reviewed: looks good." |
| bors merge- | Cancel an r+, r=, merge, or merge= |
| bors try | Run the test suite without pushing to master. |
| bors try- | Cancel a try |
| bors delegate+ | Allow the pull request author to merge their changes. |
| bors delegate=[list] | Allow the listed users to r+ this pull request's changes. |
| bors retry | Run the previous command a second time. |

This is a short collection of opinionated commands. For a full list of the commands read the [bors reference](https://bors.tech/documentation/).

</details>

### CODE REVIEW

**Review by herschel666 — APPROVED** [review]
bors merge

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            ${{ runner.OS }}-
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Create temporary .npmrc
+        run: echo '//registry.npmjs.org/:_authToken=${NPM_TOKEN}' > .npmrc
+      - name: Release
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: yarn release --yes
```
Comment (KnisterPeter) [inline .github/workflows/release.yml:38]: Shouldn't this be `yarn release:create-nightly`?

Comment (KnisterPeter) [reply .github/workflows/release.yml:38]: And why does it run on every `master` commit?
I would expect a nightly to run on a cron expression.

Comment (herschel666) [reply .github/workflows/release.yml:38]: No, we already have a nightly version running & lerna proceeds with the current type of version.

This command is for creating the `*-nightly.0`-version.

Comment (ZauberNerd) [reply .github/workflows/release.yml:38]: Cron expression might come later, first we want to test whether everything works as expeced.

**Pull request merged**